pikchr-mode.el

```diff
@@ -225,7 +225,10 @@ the block contents as BODY and its header argumenst as PARAMS."
       (insert body))
     (org-babel-eval
      (concat (org-babel-process-file-name pikchr-executable)
-             " --svg-only " (org-babel-process-file-name in-file))
+             " --svg-only "
+             (when (assq :dark-mode params) "--dark-mode ")
```

Comment:
Do we need to let Org babel know that `:dark-mode` does not take arguments? Or will it work out of the box?